patch-package
--- a/node_modules/web3-eth-accounts/node_modules/eth-lib/lib/bytes.js
+++ b/node_modules/web3-eth-accounts/node_modules/eth-lib/lib/bytes.js
@@ -4,7 +4,7 @@ const at = (bytes, index) => parseInt(bytes.slice(index * 2 + 2, index * 2 + 4),
 
 const random = bytes => {
   let rnd;
-  if (typeof window !== "undefined" && window.crypto && window.crypto.getRandomValues) rnd = window.crypto.getRandomValues(new Uint8Array(bytes));else if (typeof require !== "undefined") rnd = require("c" + "rypto").randomBytes(bytes);else throw "Safe random numbers not available.";
+  if (typeof window !== "undefined" && window.crypto && window.crypto.getRandomValues) rnd = window.crypto.getRandomValues(new Uint8Array(bytes));else if (typeof require !== "undefined") rnd = require("crypto").randomBytes(bytes);else throw "Safe random numbers not available.";
   let hex = "0x";
   for (let i = 0; i < bytes; ++i) hex += ("00" + rnd[i].toString(16)).slice(-2);
   return hex;
